src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Cmdlets/GetSettingCmdlet.cs

```diff
@@ -1,34 +1,35 @@
-﻿// -----------------------------------------------------------------------------
-// <copyright file="GetSettingsCmdlet.cs" company="Microsoft Corporation">
-//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
-// </copyright>
-// -----------------------------------------------------------------------------
-
-namespace Microsoft.WinGet.Client.Cmdlets.Cmdlets
-{
-    using System.Management.Automation;
-    using Microsoft.WinGet.Client.Common;
-    using Microsoft.WinGet.Client.Engine.Commands;
-
-    /// <summary>
-    /// Gets winget settings.
-    /// </summary>
-    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.Settings)]
-    public sealed class GetSettingsCmdlet : PSCmdlet
-    {
-        /// <summary>
-        /// Gets or sets a value indicating whether to output a string or a hashtable.
-        /// </summary>
-        [Parameter(ValueFromPipelineByPropertyName = true)]
-        public SwitchParameter AsPlainText { get; set; }
-
-        /// <summary>
-        /// Get settings.
-        /// </summary>
-        protected override void ProcessRecord()
-        {
-            var command = new CliCommand(this);
-            command.GetSettings(this.AsPlainText.ToBool());
-        }
-    }
-}
+﻿// -----------------------------------------------------------------------------
+// <copyright file="GetSettingsCmdlet.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.Cmdlets.Cmdlets
+{
+    using System.Management.Automation;
+    using Microsoft.WinGet.Client.Common;
+    using Microsoft.WinGet.Client.Engine.Commands;
+
+    /// <summary>
+    /// Gets winget settings.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.Setting)]
+    [Alias("gwgse")]
```

Comment:
We should have the old name as an alias for all of these that are changing, no?  Or were these never available in a release version of the module?

Reply:
I was in the middle of fixing this last week & will be included in my next commit